OutOfSchool/OutOfSchool.WebApi.Tests/Services/StatusServiceTests.cs

```diff
@@ -30,21 +30,30 @@ public class StatusServiceTests
     private IMapper mapper;
 
     [SetUp]
-    public void Setup()
+    public async Task Setup()
```

Comment:
(I'm curious) This `Setup()` method returns `Task` and is `async`, but I see no `await` inside the method body, and no returns. Could you provide info, why we need such change?

Reply:
Sorry. I had async method SeedDatabase before. And I forgot to change it.